### PR TITLE
Updated readme with npm commands

### PR DESCRIPTION
**Pull request by tunglinn**
Addresses comment in #144 

### Added ###
- npm commands
- a note about using wsl with npm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,60 @@ First, `npm install`. Then, run either:
...
+
+When the server is running, you can make any changes you want to the project. Reload the page in the browser to see those changes. (You may need to force reload, e.g. hold the shift key while reloading on macOS.)
+
+### Run tests
+
+```bash
+❯ npm test
+```
+
+These tests are currently for the Shiny map.
```
Comment (Eric-Arellano) [inline README.md:80]: Not quite. These tests are for our data import scripts, which are written in Javascript.

But you can get rid of this sentence either way. It's not going to be relevant in a few days when we have tests both for the map and scripts/

File: README.md [hunk 2]
```diff
@@ -54,6 +54,60 @@ First, `npm install`. Then, run either:
...
+
+```bash
+❯ npm run lint
+```
+
+### Try out a build locally
+
+You can preview what a build will look like by running `npm run build`. Then use `npm run serve-dist` to start the server.
+
+You can also run our integration tests on built dist folder with `npm run test-dist` (make sure the server is not already running).
```
Comment (Eric-Arellano) [inline README.md:105]: Technically not yet true.

**Review by Eric-Arellano — COMMENTED** [review]
Thanks! It may be worth updating the top of this file, including the Running the map app. For example, we could move the R content to the bottom of the page and call it something like "Legacy R map", since JavaScript is the future.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,66 @@ This code runs the Mandates Map app for the Parking Reform Network: https://park
...
+
+"Linting" means using tools that check for common issues that may be bugs or low code quality.
+
+```bash
+❯ npm run lint
+```
+
+### Try out a build locally
+
+You can preview what a build will look like by running `npm run build`. Then use `npm run serve-dist` to start the server. A 'build' are the files sent for production on the real site. This is slightly different from the development server run by `npm start`, which prioritizes a quick start for development.
```
Comment (tunglinn) [inline README.md:58]: Is the part about "a build" accurate? Also, is it necessary to include?

Comment (Eric-Arellano) [reply README.md:58]: Yep, it's true. And I think useful to include.

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

**Pull request merged**